CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 
+### Added
+- `opentelemetry-instrumentation-aiohttp-client` Add support for optional custom trace_configs argument.
+  ([1079](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1079))
```

Comment:
Move this `Changed` section in Unreleased.

Reply:
This is still not moved

Reply:
Hi @srikanthccv, thanks, did you mean to place it here?